Watchman.DomainModel/Settings/GetConfigurationInformationQuery.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Watchman.Cqrs;
+
+namespace Watchman.DomainModel.Settings
+{
+    public class GetConfigurationInformationQuery :IQuery<GetConfigurationInformationQueryResult>
```

Comment:
```suggestion
    public class GetConfigurationQuery : IQuery<GetConfigurationQueryResult>
```
nie ma sensu komplikować nazwy, skoro dłuższa wersja więcej nie wnosi informacji o zmiennej